cppcache/src/ThinClientRegion.cpp

```diff
@@ -49,8 +49,14 @@ namespace apache {
 namespace geode {
 namespace client {
 
-static const std::regex PREDICATE_IS_FULL_QUERY_REGEX(
-    "^\\s*(?:select|import)\\b", std::regex::icase);
+class IsFullQueryRegex {
```

Comment:
I guess I would have just don it within the `ThinClientRegion` class and made the method `private` since there is no need for it to be public (well maybe for tests but there are none).